packages/pluggableWidgets/color-picker-web/src/components/__tests__/ColorPicker.spec.tsx

```diff
@@ -11,11 +11,14 @@ describe("ColorPicker", () => {
     const colorPickerProps: ColorPickerProps = {
```

Comment:
Can't tag the lines above this one, but you should also update the `createElement` calls to JSX/component calls 😄